crates/transport_tx5/src/lib.rs

```diff
@@ -442,6 +444,7 @@ async fn evt_task(
                         continue;
                     }
                 };
+                handler.mark_peer_unresponsive(peer_url.clone());
```

Comment:
Marking peers as unresponsive also in case of a "Disconnected" event. I don't know how to test this independently from the `mark_peer_unresponsive()` call above when the send fails because they both get invoked in the test scenario that I set up.

Reply:
I don't know about this. Sometime (I hope reasonably) soon, we will actually be closing connections we don't need. I don't think it's the case that we shouldn't re-open this connection if we need to.

If the peer was online and able to successfully notify us of closing their connection then it's valid to try to connect to them again on the same peer URL I think.

Reply:
Okay, so we should rather remove this line, right?

Reply:
Removed: 40e7afd790424b6091df035fdf7f0213ceebee6e